community-operators/prometheus/prometheusoperator.0.22.2.clusterserviceversion.yaml

```diff
@@ -70,9 +70,9 @@ spec:
   - type: SingleNamespace
     supported: true
   - type: MultiNamespace
-    supported: false
-  - type: AllNamespaces
     supported: true
+  - type: AllNamespaces
+    supported: false
```

Comment:
This change doesn't seem to be related to the etcd update

Reply:
We also realized that the Prometheus Operator does not support watching all namespaces.